### ISSUE TITLE
"plugins" property of .babelrc is ignored

### ISSUE THREAD

**Issue by koss-lebedev** [prichodko/create-strv-app#9]
I'm adding `babel-plugin-module-resolver` plugin using this configuration in `.babelrc`:

```json
"plugins": [
    [
      "module-resolver",
      {
        "root": ["."],
        "alias": {
          "containers": "./src/containers"
        }
      }
    ]
  ]
```

then in `index.js` I use `import App from 'containers/App'`.

**Expected behavior**

`App` is imported and application starts

**Actual behavior**

Application fails with error:

```
'containers' should be listed in the project's dependencies. Run 'npm i -S containers' to add it  import/no-extraneous-dependencies
```

**Comment by Pave91** [comment]
I think the solution for this problem is a different configuration, cause original create-react-app doesn't use .babelrc file. I described solution in https://github.com/prichodko/create-strv-app/issues/10 (relative paths - module resolving)

**Comment by koss-lebedev** [comment]
You're right, original `create-react-app` doesn't use it, but Pavel's fork has some effort to allow merging local `.babelrc` with the one from `create-react-app`
